.changes/v3.6.0/770-features.md

```diff
@@ -0,0 +1,2 @@
+* `vcd_catalog_item` allows using `ovf_url` to upload vApp template from URL [GH-770]
+* `vcd_catalog_item` update allows changing `name` and `description`  [GH-770]
```

Comment:
Please reflect this addition in the PR's name and future merge commit message.

BTW:
```suggestion
* `vcd_catalog_item` update allows changing `name` and `description` [GH-770]
```

Reply:
fixed, PR renamed